packages/@react-spectrum/button/test/Button.test.js

```diff
@@ -273,6 +273,40 @@ describe('Button', function () {
     expect(eventDown.defaultPrevented).toBeFalsy();
   });
 
+  // is pending state
+  it('displays a spinner after a short delay when isPending prop is true', function () {
+    jest.useFakeTimers();
+    let onPressSpy = jest.fn();
+    function TestComponent() {
+      let [pending, setPending] = useState(false);
+      return (
+        <Button onPress={(pending) => { setPending(true); onPressSpy(); }} isPending={pending}>
+          Click Me
+        </Button>
+      );
+    }
+    let {getByRole} = render(<TestComponent />);
+    let button = getByRole('button');
+    expect(button).toHaveAttribute('aria-disabled', 'false');
+    fireEvent.click(button);
+    expect(button).toHaveAttribute('aria-disabled', 'true');
+    // Multiple clicks shouldn't call onPressSpy
+    fireEvent.click(button);
+    act(jest.runAllTimers);
```

Comment:
```suggestion
    act(() => {jest.advanceTimersByTime(999);});
    triggerPress(button);
    expect(onPressSpy).toHaveBeenCalledTimes(1);
    expect(button).toHaveAttribute('aria-disabled', 'true');
    expect(queryByRole('progressBar')).not.toBeInTheDocument();
    act(() => {jest.advanceTimersByTime(1);}); // 1 full second since first press
```
I know your way works, but this will match our other uses and lint won't complain about the wrong return value from runAllTimers for act should we ever make this a ts file

in addition, I'd like the delay to be in a test so that it's very obvious if we should ever change the delay and I want to test the boundary of the timer so that we know it actually waited to show the spinner